.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /coverage/
 /.nyc_output/
+package-lock.json
```

Comment:
nbd, but we have an `.npmrc` with `package-lock = false`, so you shouldn't see package locks appear?

Reply:
Oh, something weird going on with my system them, I guess...

Reply:
Since some `npm` commands will behave differently when there’s a lock file, I prefer the strategy of preventing it from being generated in the first place (in libs).

Maybe you have a `~/.npmrc` that somehow overrides it? 🤨

But either way it’s ok to _also_ have it in `.gitignore`, so don’t worry about it.